apps/sharp-tune/sharp-tune.yml

```diff
@@ -0,0 +1,11 @@
+name: Sharp-Tune
+description: 'Lightweight Cross-platform music player build upon the node using the electron framework.'

```

Comment:
No need to mention node and electron in the description. How about just `Lightweight cross-platform music player`?

Reply:
sounds great!

Reply:
Please check i have made the necessary changes .......... thanks for the advice

Reply:
@zeke i have updated the screenshot for the app as well and added a few more keywords have a look!